OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -126,9 +129,12 @@ public LaserZap(LaserZapInfo info, ProjectileArgs args, Color color)
 
 		public void Tick(World world)
 		{
+			// Position of firing actor might have changed
+			source = args.Source;
```

Comment:
Probably better to remove the commit for now, yes. It's possible some 3rd-party mods might prefer the bleed behavior of `source` not changing, at least for lasers that don't deal continous damage, so we might need another toggle property and that's out of scope for this PR.